snooty/test_postprocess.py

```diff
@@ -47,76 +52,87 @@ def test_deprecated_versions(backend: Backend) -> None:
     }
 
 
+def test_literal_includes(backend: Backend) -> None:
+    # Test a simple literally-included code block
+    page_id = FileId("literal-include.txt")
+    ast = backend.pages[page_id].ast
+    literal_include_node = ast.children[0]
+    assert isinstance(literal_include_node, n.Directive)
+    assert literal_include_node.name == "literalinclude"
+    assert len(literal_include_node.children) == 1
+
+    check_ast_testing_string(
+        literal_include_node,
+        """<directive name="literalinclude">
+<text>includes/sample_code.js</text>
+<code copyable="True" linenos="True">var str = "sample code";
+var i = 0;
+for (i = 0; i &lt; 10; i++) {
+  str += i;
+}</code>
+</directive>""",
+    )
+
+    # Test a literally-included code block with fully specified options
+    literal_include_node = ast.children[1]
+    check_ast_testing_string(
+        literal_include_node,
+        """<directive name="literalinclude" dedent="4" end-before="end example 1" language="python" start-after="start example 1">
+        <text>includes/sample_code.py</text>
+        <code copyable="True" lang="python" linenos="True">print("test dedent")</code>
```

Comment:
`copyable` and `linenos` were not specified in the `literalinclude` usage, so they actually shouldn't be appearing here! This is related to my comment above about the instantiation of `Code`.

Reply:
Since `copyable` [defaults to True](https://github.com/mongodb/snooty-parser/blob/master/snooty/parser.py#L546), I think it should show up in the default example, no? But `linenos`, I agree